lms/djangoapps/student_account/views.py

```diff
@@ -380,6 +381,37 @@ def _third_party_auth_context(request, redirect_to, tpa_hint=None):
                 context['errorMessage'] = _(unicode(msg))  # pylint: disable=translation-of-non-string
                 break
 
+        context = _update_context_for_enterprise_error_message(context, enterprise_customer)
+
+    return context
+
+
+def _update_context_for_enterprise_error_message(context, enterprise_customer=None):
```

Comment:
It would be nice if we could consolidate this with the other various functions that update the context with enterprise data and move that consolidated function to the `openedx.features.enterprise_support` module somewhere.

Reply:
I have moved this method to `utils` in `openedx->enterprise_support`.
I have tried moving most of the enterprise specific code also there but lots of tests starts failing. If moving all enterprise specific is not critical for this ticket then perhaps we can create a separate story for that.